purchase_requisition_auto_rfq/models/purchase_requisition.py

```diff
@@ -61,3 +70,70 @@ def _create_purchase_order(self, seller_id):
         po = self.env["purchase.order"].with_context(**ctx).create(vals)
         po._onchange_requisition_id()
         return po
+
+    def action_all_rfq_related_send(self):
+        self.ensure_one()
+        rfqs = self.env["purchase.order"].search([("requisition_id", "=", self.id)])
```

Comment:
If apply next suggestion, you'll benefit from Odoo's ORM caching and prefetching. I invite you to see this interesting talk which gives some info about how to benefit from those: https://www.youtube.com/watch?v=5jT5eoCZH6U

For now, let's just say that this is more efficient:

```suggestion
        rfqs = self.purchase_ids
```